lib/matplotlib/backend_bases.py

```diff
@@ -2693,6 +2693,10 @@ def set_message(self, s):
 
     def back(self, *args):
         """Move back up the view lim stack."""
+        if args:
```

Comment:
https://github.com/matplotlib/matplotlib/pull/16095 gives you `_delete_parameter(version, "args")` and `_delete_parameter(version, "kwargs)`.

Reply:
If that gets in first, I'll adapt, but wouldn't wait for this. BTW, I've approved #16063.

Reply:
You can also steal the implementation of _delete_parameter if you want.

Reply:
I'll leave that to you :smile: